src/zfm20/zfm20.cxx

```diff
@@ -540,10 +439,10 @@ uint8_t ZFM20::deleteDB()
   return rPkt[9];
 }
 
-uint8_t ZFM20::search(int slot, uint16_t *id, uint16_t *score)
+uint8_t ZFM20::search(int slot, uint16_t &id, uint16_t &score)
 {
```

Comment:
>  IMO, this is easier (at least in python) since it returns a list (example search(1) = [0, 1, 80])...

I'm confused... search accepts 3 args and returns a uint8_t... So I'm not sure how you call search(1), and how you get a tuple back...?